src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Windows.cs

```diff
@@ -107,6 +116,14 @@ public static SecurityStatusPal InitializeSecurityContext(ref SafeFreeCredential
         }
 
         public static SafeFreeCredentials AcquireCredentialsHandle(X509Certificate? certificate, SslProtocols protocols, EncryptionPolicy policy, bool isServer)
+        {
+            // New crypto API does not allow to force NULL encryption.
+            return !UseNewCryptoApi || policy == EncryptionPolicy.NoEncryption ?
+                        AcquireCredentialsHandleLegcy(certificate, protocols, policy, isServer) :
```

Comment:
AcquireCredentialsHandleLegcy => AcquireCredentialsHandleLegacy

Also, could we use a different term than Legacy?  Is it PreWindows101809 vs PostWindows101809?

Reply:
Would it make sense to name them by the structure they use? The name is similar so even with that I found it confusing but that is really the main difference. The structures them selves have version field and it is v4 for "legacy" and v5 for the "new" one.

Reply:
> Would it make sense to name them by the structure they use? 

Sounds good.

Reply:
I made the change and added comment to each.